app/client/sagas/authSagas.js

```diff
@@ -197,10 +206,27 @@ function* watchRegisterRequest() {
 function* activate({activation_token}) {
   try {
     const activated_account = yield call(activateAPI, activation_token);
-    yield put({type: ACTIVATE_SUCCESS, activated_account});
-    yield put(createLoginSuccessObject(activated_account));
-    yield call(storeSessionToken, activated_account.auth_token );
-  } catch (error) {
+
```

Comment:
Added TFA handling to the activate flow.

This means that a user who receives an email with login link, will automatically go to TFA screen. An alternative was to call login flow api, but that felt sub par as would be an extra api call to the flow (which could be slow on poor connections)